cico_run_tests.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+# This file is supposed to be executed by CICO build (ci.centos.org/view/Devtools/)
+#   1. Run tests for fabric8-planner
+#   2. Create snapshot with fabric8-ui's master and fabric8-planner's current changes
+
+# Show command before executing
+set -x
+
+# Exit on error
+set -e
+
+# Source environment variables of the jenkins slave
+# that might interest this worker.
+if [ -e "jenkins-env" ]; then
+  cat jenkins-env \
+    | grep -E "(JENKINS_URL|DEVSHIFT_USERNAME|DEVSHIFT_PASSWORD|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
+    | sed 's/^/export /g' \
+    > /tmp/jenkins-env
+  source /tmp/jenkins-env
+fi
+
+# We need to disable selinux for now, XXX
+/usr/sbin/setenforce 0
+
+# Get all the deps in
+yum -y install docker
+service docker start
+
+# Build builder image
+cp /tmp/jenkins-env .
+
+mkdir -p fabric8-ui-dist
+# Build fabric8-planner image
+docker build -t fabric8-planner-builder .
+# User root is required to run webdriver-manager update.
+# This shouldn't be a problem for CI containers
+# Chrome crashes on low size of /dev/shm. We need the --shm-size=256m flag.
+CID=$(docker run --detach=true \
+    --shm-size=256m \
+    -v $(pwd)/fabric8-ui-dist:/home/fabric8/fabric8-planner/fabric8-ui-dist:Z \
```

Comment:
Please change to: `-v $(pwd)/fabric8-ui-dist:/home/fabric8/fabric8-planner/dist:Z \`

Reply:
hi @kwk while we were discussing I forgot, this `dist` we want on the host is not from planner. It is from fabric8-ui directory that we have cloned inside the container. So, I do not see other option than to copy it ATM

Reply:
@pranavgore09 you could simply mount two directories into the container :)

Reply:
Are we talking about this: https://github.com/fabric8-ui/fabric8-planner/blob/master/scripts/bootstrap.sh#L102

Reply:
Please clarify the purpose, so that we're on the same page. Something seems off here.